module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,15 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string> wordsList) {
+    wordsList.sort([](std::string a, std::string b) {
+        if (a.size() == b.size()) {
+            return std::lexicographical_compare(a.begin(), a.end(), b.begin(), b.end());
+        } else {
+            return a.size() < b.size();
+        }
```

Comment:
You do not need else here. Return end function,so you won't reach the remaining code when if statement is true.

Reply:
+1